page-template-cluster.php

```diff
@@ -147,6 +139,50 @@
 	</section>
 	<?php endif; ?>
 	<!-- End Research -->
+	<!-- Start Academics -->
+	<?php if ( is_array( $cluster_programs ) && ! empty( $cluster_programs ) ) : ?>
+	<section class="jumbotron bg-inverse text-inverse" aria-labelledby="research-programs">
```

Comment:
`mb-0` can be added to these `jumbotron`s to remove the gap between subsequent sections.

Reply:
Updated.